### PR TITLE
Guide Script add, and update script

### PR DESCRIPTION
**Pull request by ZacharyWills**

### CODE REVIEW

**Comment by ZacharyWills** [comment]
for #26

File: docker/HelloNGEN.sh [hunk 1]
```diff
@@ -51,7 +51,11 @@ select opt in ngen-parallel ngen-serial bash quit; do
       break
       ;;
     bash)
+<<<<<<< Updated upstream
```
Comment (benlee0423) [inline docker/HelloNGEN.sh:54]: line 54 to 58 looks strange. We need to clean up those arrows and echo?

**Review by benlee0423 — COMMENTED** [review]
Looks good to me, but I can't test it locally because of machine resource issue in my personal laptop.

**Comment by ZacharyWills** [comment]
Probably need to rerun the build since it's passing now with the upstream fixes.

**Comment by hellkite500** [comment]
#50 removes the patch causing the build failure, now that it is merged this PR test should pass.

**Review by arpita0911patel — APPROVED** [review]
Changes looks good. Approving.

**Pull request merged**